packages/components/molecules/f-alert/src/components/Alert.vue

```diff
@@ -155,14 +154,12 @@ $alert-borderRadius: $border-radius;
     }
 
     .c-alert-icon {
+        min-width: 20px;
```

Comment:
would setting min-width the same as width make it redundant?

Reply:
No, it seemed to be required to stop the icon from shrinking

Reply:
ok makes sense